### DISCUSSION TITLE
Updating Hugo and Doks

### DISCUSSION THREAD

**Discussion by technosprout** [thuliteio/doks#1053] [General]
Newbie here

I am on a Mac system (intel)
Need to update hugo/doks/hyas

1. How do you update the HUGO portion of the code with npm?
2. How do you update the DOKS portion of the code with  npm?
3. How do you update the HYAS portion of code with npm?

Thanks

**Comment by h-enk** [comment]
All dependencies are managed centrally in `package.json` (for more details, see [dependencies](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#dependencies))

To update Hugo, bump the version in the [`otherDependencies`](https://github.com/h-enk/doks/blob/master/package.json#L69) section of `package.json`, then run `npm install` from the command line.

To check for outdated packages, run `npm outdated`. To update packages, run `npm update`,

See also [How to Update](https://getdoks.org/docs/help/how-to-update/)
